project-generator/src/main/java/com/redhat/ipaas/project/converter/DefaultProjectGenerator.java

```diff
@@ -161,6 +162,18 @@ public DefaultProjectGenerator(ConnectorCatalog connectorCatalog, ProjectGenerat
                     continue;
                 }
 
+                if ( "mapper".equals(step.getStepKind()) ) {
```

Comment:
Couldn't we introduce a StepKinds.MAPPER (like there is a StepKinds.ENDPOINT) ? If the mapper has a special meaning and needs special treatment, then is should be treated as a special step, too. Actually I would even make that an enum.